components/Dialogs/src/MDCAlertController.h

```diff
@@ -291,15 +300,60 @@
  */
 - (void)addAction:(nonnull MDCAlertAction *)action;
 
-/** MDCAlertController handles its own transitioning delegate. */
-- (void)setTransitioningDelegate:
-    (_Nullable id<UIViewControllerTransitioningDelegate>)transitioningDelegate NS_UNAVAILABLE;
+/** Content alignment for Alert actions. */
```

Comment:
Please file a follow-up task to replace this enum with UIControlContentHorizontalAlignment once we drop support for iOS 10 and include it as a `TODO(<url>)` statement in the docs here.

Reply:
Filed https://github.com/material-components/material-components-ios/issues/9891 and updated the PR.